.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,19 @@
+- id: buf-breaking
+  name: buf breaking
+  language: golang
+  entry: buf breaking
+  # match files ending in .proto,
+  files: &protofiles '[^/]+\.proto$'
```

Comment:
As opposed to using anchors, I'd prefer all three `files` keys are just `files: '[^/]+\.proto$'`.

Also, shouldn't this be `[^\/]+\.proto$`, and even moreso, is this regex correct? This won't work on Windows, as an example - we could probably simplify this down to just `\.proto$`.